### ISSUE TITLE
Crash caused by :  java.net.UnknownHostException Unable to resolve host "api.instabug.com"

### ISSUE THREAD

**Issue by guptakavya** [Instabug/Instabug-React-Native#536]
```
io.reactivex.plugins.RxJavaPlugins.onError (RxJavaPlugins.java:367)
io.reactivex.internal.operators.completable.CompletableMergeIterable$MergeCompletableObserver.onError (CompletableMergeIterable.java:124)
io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.onError (CompletableSubscribeOn.java:74)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.onError (CompletableConcatArray.java:60)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.onError (CompletableConcatArray.java:60)
io.reactivex.internal.operators.completable.CompletablePeek$CompletableObserverImplementation.onError (CompletablePeek.java:96)
io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable$IgnoreObservable.onError (ObservableIgnoreElementsaCompletable.java:61)
io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.onError (ObservableRetryPredicate.java:78)
io.reactivex.internal.operators.observable.ObservableCreate$CreateEmitter.tryOnError (ObservableCreate.java:85)
io.reactivex.internal.operators.observable.ObservableCreate$CreateEmitter.onError (ObservableCreate.java:73)
com.instabug.library.network.NetworkManager$a.subscribe (NetworkManager.java:104)
io.reactivex.internal.operators.observable.ObservableCreate.subscribeActual (ObservableCreate.java:40)
io.reactivex.Observable.subscribe (Observable.java:12025)
io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.subscribeNext (ObservableRetryPredicate.java:111)
io.reactivex.internal.operators.observable.ObservableRetryPredicate.subscribeActual (ObservableRetryPredicate.java:41)
io.reactivex.Observable.subscribe (Observable.java:12025)
io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable.subscribeActual (ObservableIgnoreElementsCompletable.java:31)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletablePeek.subscribeActual (CompletablePeek.java:51)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.next (CompletableConcatArray.java:89)
io.reactivex.internal.operators.completable.CompletableConcatArray.subscribeActual (CompletableConcatArray.java:33)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.next (CompletableConcatArray.java:89)
io.reactivex.internal.operators.completable.CompletableConcatArray.subscribeActual (CompletableConcatArray.java:33)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run (CompletableSubscribeOn.java:64)
io.reactivex.Scheduler$DisposeTask.run (Scheduler.java:571)
io.reactivex.internal.schedulers.ScheduledRunnable.run (ScheduledRunnable.java:66)
io.reactivex.internal.schedulers.ScheduledRunnable.call (ScheduledRunnable.java:57)
java.util.concurrent.FutureTask.run (FutureTask.java:266)
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run (ScheduledThreadPoolExecutor.java:301)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
java.lang.Thread.run (Thread.java:919)
```

```
Caused by java.net.UnknownHostException
Unable to resolve host "api.instabug.com": No address associated with hostname
com.instabug.library.network.NetworkManager.connectWithNormalType

java.net.Inet6AddressImpl.lookupHostByName (Inet6AddressImpl.java:124)
java.net.Inet6AddressImpl.lookupAllHostAddr (Inet6AddressImpl.java:103)
java.net.InetAddress.getAllByName (InetAddress.java:1152)
com.android.okhttp.Dns$1.lookup (Dns.java:41)
com.android.okhttp.internal.http.RouteSelector.resetNextInetSocketAddress (RouteSelector.java:178)
com.android.okhttp.internal.http.RouteSelector.nextProxy (RouteSelector.java:144)
com.android.okhttp.internal.http.RouteSelector.next (RouteSelector.java:86)
com.android.okhttp.internal.http.StreamAllocation.findConnection (StreamAllocation.java:192)
com.android.okhttp.internal.http.StreamAllocation.findHealthyConnection (StreamAllocation.java:144)
com.android.okhttp.internal.http.StreamAllocation.newStream (StreamAllocation.java:106)
com.android.okhttp.internal.http.HttpEngine.connect (HttpEngine.java:400)
com.android.okhttp.internal.http.HttpEngine.sendRequest (HttpEngine.java:333)
com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute (HttpURLConnectionImpl.java:483)
com.android.okhttp.internal.huc.HttpURLConnectionImpl.connect (HttpURLConnectionImpl.java:135)
com.android.okhttp.internal.huc.HttpURLConnectionImpl.getOutputStream (HttpURLConnectionImpl.java:266)
com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getOutputStream (DelegatingHttpsURLConnection.java:219)
com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getOutputStream (HttpsURLConnectionImpl.java:30)
com.instabug.library.network.NetworkManager.connectWithNormalType (NetworkManager.java:8)
com.instabug.library.network.NetworkManager.access$100 (NetworkManager.java:1)
com.instabug.library.network.NetworkManager$a.subscribe (NetworkManager.java:20)
io.reactivex.internal.operators.observable.ObservableCreate.subscribeActual (ObservableCreate.java:40)
io.reactivex.Observable.subscribe (Observable.java:12025)
io.reactivex.internal.operators.observable.ObservableRetryPredicate$RepeatObserver.subscribeNext (ObservableRetryPredicate.java:111)
io.reactivex.internal.operators.observable.ObservableRetryPredicate.subscribeActual (ObservableRetryPredicate.java:41)
io.reactivex.Observable.subscribe (Observable.java:12025)
io.reactivex.internal.operators.observable.ObservableIgnoreElementsCompletable.subscribeActual (ObservableIgnoreElementsCompletable.java:31)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletablePeek.subscribeActual (CompletablePeek.java:51)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.next (CompletableConcatArray.java:89)
io.reactivex.internal.operators.completable.CompletableConcatArray.subscribeActual (CompletableConcatArray.java:33)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableConcatArray$ConcatInnerObserver.next (CompletableConcatArray.java:89)
io.reactivex.internal.operators.completable.CompletableConcatArray.subscribeActual (CompletableConcatArray.java:33)
io.reactivex.Completable.subscribe (Completable.java:1919)
io.reactivex.internal.operators.completable.CompletableSubscribeOn$SubscribeOnObserver.run (CompletableSubscribeOn.java:64)
io.reactivex.Scheduler$DisposeTask.run (Scheduler.java:571)
io.reactivex.internal.schedulers.ScheduledRunnable.run (ScheduledRunnable.java:66)
io.reactivex.internal.schedulers.ScheduledRunnable.call (ScheduledRunnable.java:57)
java.util.concurrent.FutureTask.run (FutureTask.java:266)
java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run (ScheduledThreadPoolExecutor.java:301)
java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1167)
java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:641)
java.lang.Thread.run (Thread.java:919)
```

### Expected Behavior
No crash

### Actual Behavior
The following crash occurrence : 
`NetworkManager.java line 8
com.instabug.library.network.NetworkManager.connectWithNormalType
`

### SDK Version

```
'com.instabug.library:instabug:9.1.6'
'com.instabug.library:instabug-with-okhttp-interceptor:9.1.6'
```

### React Native, iOS and Android Versions
Android OS 10, 8 and 9. 

### Device Model

Samsung, Google and others.

**Comment by guptakavya** [comment]
I am assuming from this issue report here : #496 that this might have. been fixed in v9.1.8? Confirming about the same?

**Comment by AliAbdelfattah** [comment]
@guptakavya, yes this issue was fixed in v9.1.8
